docs/rollout/active-hours.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Active Hours Metric
+layout: rollout
+---
+
+Choosing a good metric to track adoption is difficult. We recommend "Active Hours". Active Hours tracks during how many calendar hours a user was active in Tilt. E.g. if Alice used Tilt at 1:03PM, 1:15PM, 3:07PM and 4:21PM, Alice had 3 Active Hours (1PM, 3PM and 4PM). We recommend it because:
```

Comment:
```suggestion
Choosing a good metric to track adoption is difficult. We recommend "Active Hours". Active Hours tracks during how many clock hours a user was active in Tilt. E.g. if Alice used Tilt at 1:03PM, 1:15PM, 3:07PM and 4:21PM, Alice had 3 Active Hours (1PM, 3PM and 4PM). We recommend it because:
```